src/user-guide/writing-workflows/scheduling.rst

```diff
@@ -1938,6 +1938,18 @@ Note the last line of the graph ``c | r => d`` allows the graph to
 continue on to ``d`` regardless of the path taken. This is an :term:`artificial
 dependency`.
 
+This is a simple example of a "switch" pattern, the task ``b`` being the switch
+in this case, the ``succeeded`` / ``failed`` outputs deciding which pathway
+through the graph the workflow will follow. We can use outputs besides
+``succeeded`` / ``failed`` to achieve this and can have any number of branches,
+see the
+:ref:`three-way switch example <user_guide.graph_branching.three_way_switch>`
+for more details.
```

Comment:
Link into later "custom outputs" example which implements a three-way switch (making it clear graph branching is not limit to two branches) and custom outputs (making it clear graph branching is not limited to succeeded/failed).